lib/daru.rb

```diff
@@ -58,11 +61,21 @@ def create_has_library(library)
         class_variable_get(cv)
       end
     end
+
+    def plotting_library= lib
+      case lib
+      when :gruff, :nyaplot
+        @plotting_library = lib
+      else
+        raise ArgumentError, "Unsupported library #{lib}"
+      end
+    end
```

Comment:
> Is there no case when you want Gruff graphs (pretty & static) and Nyaplot graphs (unstable yet dynamic) at the same presentation? And what if we'll add one more graphing library? Always select one and only one? Switch this setting on any action? 

I have provided an option `#plotting_library` with dataframes and vectors also. So one can use, for example `dv.plotting_library=:gruff` if he/she wants to use gruff for this particular vector. When we set the library using `Daru.plotting_library`, it means all the vectors or dataframes created after that will follow this plotting library for the plots but you can change setting for particular vectors and dataframes.

So what do you say? Is it still a problem?